NameGame/NameGame/APIService.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  APIService.swift
+//  NameGame
+//
+//  Created by Tammy Le on 6/12/17.
+//  Copyright © 2017 WillowTree Apps. All rights reserved.
+//
+
+import Foundation
+
+/**
+    Reads and models data from online API.
+ */
+
+class APIService {
+    
+    static let manager = APIService()
+    
+    // Url for user records API
+    let apiString: String = "https://willowtreeapps.com/api/v1.0/profiles/"
```

Comment:
We shouldn't need this URL outside our API service. So, let's mark it as private. Also, I think we should add this to an `EndPoint` struct that is internal to the service.

```
private struct Endpoint {
    let userRecords = URL(string: "https://willowtreeapps.com/api/v1.0/profiles/")!
}
```

You will notice that I used force unwrapping here. This is considered acceptable in this case because we want the app to fail early if we have a malformed URL in our `EndPoint` struct.

You can then use your new endpoint in the loadData method like this.

```
let task = URLSesson.shared.dataTask(with: Endpoint.userRecords) { ...
```